frontend/src/lib/SessionInfo.ts

```diff
@@ -31,7 +32,11 @@ export class SessionInfo {
   public readonly authorEmail: string
 
   // Fields that can change during the lifetime of a session.
-  public commandLine: string[] = []
+  public args: string[] = []
+
+  // This should not change, but since it is handled by the NewReport message
+  // we need to declare it here.
+  public commandLine?: string = undefined
```

Comment:
I don't like this much. This should be `readonly`, but if it so, then it needs to be passed in the "initialize" message

Reply:
Talked offline. Move to Initialize message.